R-package/tests/testthat/test_basic.R

```diff
@@ -148,18 +150,21 @@ test_that("training continuation works", {
   bst1 <- xgb.train(param, dtrain, nrounds = 2, watchlist)
   # continue for two more:
   bst2 <- xgb.train(param, dtrain, nrounds = 2, watchlist, xgb_model = bst1)
-  expect_equal(bst$raw, bst2$raw)
+  if (!windows_flag)
+    expect_equal(bst$raw, bst2$raw)
```

Comment:
Actually if you want to skip this test, you should also skip it on Sparc Solaris.